tests/validation/README.md

```diff
@@ -0,0 +1,90 @@
+
+| Switch | Long switch       | Variable type      | Argument value         | Default                        | Examples                   |
+| ------ | ----------------- | ------------------ | ---------------------- | ------------------------------ | -------------------------- |
+| `-H`   | `--help`          | N/A                | empty (no argument)    | N/A                            | N/A                        |
+| `-b`   | `--file_name`     | `static char[128]` | Input/Output file name | `""` (empty)                   | `"in.yuv"` / `"out.yuv"`   |
+| `-w`   | `--width`         | `uint32_t`         | Video width [pixels]   | `1920`                         | `1920`                     |
+| `-h`   | `--height`        | `uint32_t`         | Video height [pixels]  | `1080`                         | `1080`                     |
+| `-f`   | `--fps`           | `double`           | Frames per second      | `30.0`                         | `25`                       |
+| `-x`   | `--pix_fmt`       | `char[32]`         | Video pixel format     | `"yuv422p10le"`                | `"yuv444p"`                |
+| `-r`   | `--rcv_ip`        | `char[253]`        | Receiver IP address    | `"127.0.0.1"`                  | `"192.168.96.1"`           |
+| `-i`   | `--rcv_port`      | `char[5]`          | Receiver port number   | `9001`                         | `9001`                     |
+| `-s`   | `--send_ip`       | `char[253]`        | Sender IP address      | `"127.0.0.1"`                  | `"192.168.96.2"`           |
+| `-p`   | `--send_port`     | `char[5]`          | Sender port number     | `9001`                         | `9002`                     |
+| `-o`   | `--protocol_type` | `char[32]`         | Type of protocol       | `"auto"`                       | `"memif"`                  |
+| `-t`   | `--payload_type`  | `char[32]`         | Type of payload        | `"st20"`                       | `"rdma"`/`"st22"`/`"st30"` |
+| `-k`   | `--socketpath`    | `char[108]`        | Socket path            | `"/run/mcm/mcm_rx_memif.sock"` | `"/other/path.sock"`       |
+| `-d`   | `--interfaceid`   | `uint32_t`         | ID of an interface     | `0`                            | `1`                        |
+| `-l`   | `--loop`          | `bool`             | Whether to loop or not | `0` (false)                    | `1` (true)                 |
+
+### Usage examples
+
+Sender:
+- `sudo media_proxy -d 0000:c0:11.1 -t 9992`
+- `sudo MCM_MEDIA_PROXY_PORT=9992 ./sender -w 1920 -h 1080 -f 30 -x yuv422p10le" -s 192.168.96.1 -r 192.168.96.2 -i 9991 -b input.yuv`
+
+Receiver:
+- `sudo media_proxy -d 0000:c0:11.0 -t 9991`
+- `sudo MCM_MEDIA_PROXY_PORT=9991 ./recver -w 1920 -h 1080 -f 30 -x yuv422p10le" -s 192.168.96.2 -p 9992 -b output.yuv`
+
+## Validation helpers
+
+An aside part with help information for validators.
+
+### Media Proxy hugepages requirement
+
+To meet Media Proxy hugepages requirement, as super user, add a number >10 to each of the files:
```

Comment:
Can you explain this `>10` requirement?

Reply:
I'll be honest, I have no idea how many are needed for this to run properly. As far as I remember, two was not enough.

Reply:
Please verify with @Mionsz, @tszumski, and @Sakoram.

Reply:
Here [MTL docs](https://github.com/OpenVisualCloud/Media-Transport-Library/blob/main/doc/run.md#4-setup-hugepage) recommends 4 GB. The actual needed amount depends from size of video frame and number of streams. 

Your recommendation of 10 pages of 2 MB is irrelevant, only 10 pages of 1 GB will make any difference. You can as well recommend to allocate only 1 GB pages. 

Another topic is that some time ago I noticed a problem when the allocating both page types, the MTL configuration then took a very long time. If nothing has changed, I recommend to allocate one type of huge pages. for example 8 x 1GB or 4096 x 2MB

Reply:
I reconsidered the number of each kind of hugepages based on the documentation provided and landed on hard recommending 8 GB. We usually have enough memory present on the systems to allow us to make 4 GB of each type. Additionally, a note was added with a link from Marek.

---

If you allocate only, say 2 MB hugepages, you will still be prompted with
```text
EAL: No free 1048576 kB hugepages reported on node 0
EAL: No free 1048576 kB hugepages reported on node 1
```
From my past experience, it makes the test fail. And adding the 1 GB hugepages to the mix fixes the issue.